agent/consul/status_endpoint.go

```diff
@@ -55,8 +55,16 @@ func (s *Status) Peers(args *structs.DCSpecificRequest, reply *[]string) error {
 	return nil
 }
 
-// Used by Autopilot to query the raft stats of the local server.
-func (s *Status) RaftStats(args struct{}, reply *structs.RaftStats) error {
+// EmptyReadRequest implements the interface used by middleware.RequestRecorder
+// to communicate properties of requests.
+type EmptyReadRequest struct{}
```

Comment:
i did consider throwing this into the `structs.go` file but here's some reasons against:

* we only care about `IsRead()` 
* not really sure what the immediate future of `RPCInfo` is

Reply:
This seems like a reasonable place to keep it until it's needed by other packages